### ISSUE TITLE
FontAwesomeAnimatableView

### ISSUE THREAD

**Issue by JakeLin** [IBAnimatable/IBAnimatable#104]
Support Font Awesome to draw icons, some logic can automatically generated by a script like https://github.com/JakeLin/SwiftWeather/blob/master/SwiftWeather/WeatherIcon.swift

Refer to the discussion on https://github.com/JakeLin/IBAnimatable/issues/101

**Comment by lexrus** [comment]
Oh no. I were just kidding.
Do you guys really think this is necessary? No one expecting a new Three20.
We should focus on some primary goals and do the best.

**Comment by JakeLin** [comment]
@lexrus I added as nice-to-have, if we do it it will be in a pod's sub spec like a plugin, don't want to put any asset in the lib yet.

**Comment by TheDarkCode** [comment]
I was just adding FontAwesome 4.6.3 into my framework, I can make another PR with this in the Swift3 branch.

**Comment by JakeLin** [comment]
@TheDarkCode I still think that should be a plugin sits on different repo because it requires font assets. It will depends on IBAnimatable. It can also depend on https://github.com/dzenbot/Iconic to generate the assets. The user can specify the sub-pod in Cocoapods to use this plugin. How do you think?

**Comment by tbaranes** [comment]
Since IBAnimatable is growing fast and we will start to have related plugins, what about creating an organisation to regroup all the plugins and the main library together? Otherwise, we can also just linked all the various related project in the README, but it's a bit less pro.

**Comment by TheDarkCode** [comment]
@tbaranes I have my Ahtau framework library that I was considering open sourcing. It has full coverage of a wide range of extensions and subroutines. I already incorporate Animatable in it. @JakeLin might disapprove of the tabs instead of spaces. However, everything together with combined efforts would make for one of the most comprehensive libraries on the planet.

**Comment by JakeLin** [comment]
@tbaranes good idea, can you help me with this? We need to transfer the main project to the organisation as well.

**Comment by JakeLin** [comment]
@TheDarkCode I know Ahtau will be the best framework on the planet, while IBAnimatble will still focus on Interface Builder stuff.

**Comment by tbaranes** [comment]
@JakeLin Sure, I never did it before, but I'm interesting to learn how is that working. I will be busy this weekend, but I will take a look next week!
@TheDarkCode Just curious, what is it? Jake already told me that you are working on a framework... but didn't get all the details / specifications? If there any private stuff you don't want to talk in public, we can continue by email :)

**Comment by TheDarkCode** [comment]
@tbaranes Im going to add you to a Quip in the morning when I get some time. Been a long day. Working on a grant application.
